Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Payment.hs

```diff
@@ -284,7 +284,9 @@ processNotification notificationId notificationStatus respCode respMessage fromW
     case notificationStatus of
       Juspay.NOTIFICATION_FAILURE -> do
         --- here based on notification status failed update driver fee to payment_overdue and reccuring invoice----
-        let isRetryEligibleError = maybe False (`elem` transporterConfig.notificationRetryEligibleErrorCodes) respCode
+        getNotificationScheduler <- SLDriverFee.getNotificationSchedulerKey driverFee.startTime driverFee.endTime
```

Comment:
should we call it `isNotificationSchedulerRunning` instead of `getNotificationSchedulerKey`